src/pages/Domains/index.tsx

```diff
@@ -57,21 +57,18 @@ interface DomainsProps {
 }
 
 function Domains({ address }: DomainsProps) {
-  const { data, loading } = useGetDomainsQuery({
-    variables: {
-      address: address.toLowerCase(),
-    },
-  });
+  const { emptyDomainArray, domainArray, loading } = useDomainsData(address);
```

Comment:
This should not be here. The number of calls to Nimi API is `number of domains * 1` and that's not very ideal on slow networks. We need to re-populate the fields when the user wishes to edit/create Nimi. 

You lazy-fetch the data from Nimi about each Domain in the `ENSNameCard` and add a tiny loader to the Card.

Reply:
So I made this decision because in the future well have two separate sections and theyll need to be fetched beforehand...This is more future proof once we have designs and it doesn't slow it that much...
<img width="385" alt="Screenshot 2022-09-21 at 15 43 22" src="https://user-images.githubusercontent.com/33226956/191520183-ba4f2e67-4fab-49db-ba6f-508034fc3425.png">